vkcs/networking/resource_subnet.go

```diff
@@ -115,7 +115,7 @@ func ResourceNetworkingSubnet() *schema.Resource {
 				Optional:      true,
 				ForceNew:      false,
 				Computed:      true,
-				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default gateway of `.1` to be used. Changing this updates the gateway IP of the existing subnet.",
+				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default gateway of `.1` to be used for the Neutron SDN, but the gateway does not have a default value for the Sprut SDN. Changing this updates the gateway IP of the existing subnet.",
```

Comment:
```suggestion
				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default value to be used. Changing this updates the gateway IP of the existing subnet.",
```

Reply:
To be mentioned, if I remember correctly, we cannot edit gateway IP in Sprut SDN.

Reply:
fix